Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -246,6 +247,21 @@ class AgendaDetailFragment : Fragment() {
                 scheduleDetail.listRow.talkDescription.getHtmlFormattedSpanned()
 
         tv_agenda_detail_description.movementMethod = LinkMovementMethod.getInstance()
+        tv_agenda_detail_shareText.text = resources.getString(R.string.sharing_twitter_title)
+        imgv_twitter.setOnClickListener {
+            val twitter = viewModel.getSpeaker(scheduleDetail.listRow.primarySpeakerName)?.socialProfiles?.get("twitter")
+            var twitterVal = ""
+            if(twitter.isNullOrEmpty())
+                twitterVal = scheduleDetail.listRow.primarySpeakerName
+            else
+                twitterVal = "@"+twitter
+
+
+            val tweetUrl = "https://twitter.com/intent/tweet?text=I really enjoyed this %23droidconbos talk \"" + scheduleDetail.listRow.talkTitle + "\" by " + twitterVal
```

Comment:
one thought @GiorgioNatili and I had was maybe we could add https://www.droidcon-boston.com/agenda/ to the tweet text?  Then anyone viewing the tweet could jump directly to the agenda to learn more about the talk.

if it ends up being very time consuming, don't worry about it